src/postprocess/band_structure.jl

```diff
@@ -21,14 +21,14 @@ function high_symmetry_kpath(model; kline_density=20)
 end
 
 function compute_bands(basis, ρ, kcoords, n_bands;
-                       eigensolver=lobpcg_hyper, tol=1e-5, show_progress=true, kwargs...)
+                       eigensolver=lobpcg_hyper, tol=1e-3, show_progress=true, kwargs...)
     # Create basis with new kpoints, where we cheat by using any symmetry operations.
     ksymops = [[(Mat3{Int}(I), Vec3(zeros(3)))] for _ in 1:length(kcoords)]
     bs_basis = PlaneWaveBasis(basis, kcoords, ksymops)
     ham = Hamiltonian(bs_basis; ρ=ρ)
 
     band_data = diagonalize_all_kblocks(eigensolver, ham, n_bands + 3;
-                                        n_conv_check=n_bands, interpolate_kpoints=false,
+                                        n_conv_check=n_bands,
```

Comment:
I had trouble when this was on that band structures could give rise to very strange kinks (because it is not guaranteed that when going from one branch to another interpolation makes sense). Notice that this is *not* the case for silicon. There it is perfectly reasonable to switch it on.

Reply:
Really? Remember on what system? Tol is pretty tight and should directly translate to a bound on the eigenvalues, so there might be another bug there.

Reply:
It's not a bug. It found true eigenvalues, but it missed some in the middle. The problem is that using the interpolation across kpath branches is just not a very good guess. Could have been Aluminium or something like that, but I don't remember.

Reply:
Then yeah it's a bug, the eigensolver is not supposed to miss eigenvalues (the same could happen in a SCF for instance). I'd be very interested in a repro.

Reply:
Uff ... I don't remember ... was a long time ago. But ok switch it off and then we see.